cypress/e2e/14-mapping.cy.ts

```diff
@@ -34,4 +36,265 @@ describe('Data mapping', () => {
 		ndv.actions.mapDataFromHeader(2, 'value');
 		ndv.getters.inlineExpressionEditorInput().should('have.text', '{{ $json.timestamp }} {{ $json["Readable date"] }}');
 	});
+
+	it('maps expressions from table json, and resolves value based on hover', () => {
+		cy.fixture('Test_workflow-actions_paste-data.json').then((data) => {
+			cy.get('body').paste(JSON.stringify(data));
+		});
+		canvasNode.actions.openNode('Schedule Trigger');
+		ndv.actions.setPinnedData([
+			{
+				input: [
+					{
+						"hello world": {
+							count: 0,
+						},
+					}
+				]
+			},
+			{
+				input: [
+					{
+						"hello world": {
+							count: 1,
+						}
+					}
+				]
+			},
+		]);
+		ndv.actions.close();
+
+		canvasNode.actions.openNode('Set');
+		ndv.actions.switchInputMode('Table');
+		ndv.getters.inputDataContainer().get('table', { timeout: 10000 }).should('exist');
+
+		ndv.getters.nodeParameters().find('input[placeholder*="Add Value"]').click();
```

Comment:
Nit: Can we add test id's or use getters in order to avoid such selectors in tests?

Reply:
let me try